rholang/src/test/scala/coop/rchain/rholang/interpreter/matcher/StreamTSpec.scala

```diff
@@ -110,7 +107,7 @@ class StreamTSpec extends FlatSpec with Matchers {
 }
 
 class StreamTLawsSpec
-    extends CatsSuite
```

Comment:
I assume `CatsSuite` is no longer available. Have you checked if you can get it from https://typelevel.org/cats-testkit-scalatest/ instead?

Reply:
I actually tried `cats-testkit-scalatest`. I don't use it because the `cats-testkit-scalatest` use the latest version `scalatest==3.1.0` which has a really great change(see http://www.scalatest.org/release_notes/3.1.0). All the spec names are changed. For example, `FunSuite ` -> `AnyFunSuite`. And `GeneratorDrivenPropertyChecks` is moved to `scalatestplus`(see https://github.com/scalatest/scalatest/issues/1735) and the behaviour is changed.

I looked around the whole project and there is only one place used `CatsSuite` and many other
places used `GeneratorDrivenPropertyChecks`.

I would prefer a safer way to upgrade `cats` without touching a lot of tests.

Reply:
Okay, fair enough.

Reply:
We can ditch `CatsSuite` I think. Please review https://github.com/typelevel/cats/pull/2806